content/docs/guides/ha_scale/perf-scale.md

```diff
@@ -108,8 +76,39 @@ There is not much difference between pod-rollout time and stable time in terms o
 
 By looking at the collected metrics, in our testing scenarios, resource consumption of data plane sidecars does not change much along with the mesh size. At stable time, the average CPU usage is close to 0 and average memory usage is around 50MB. CPU usage is related to request processing and memory usage is related the size of Envoy configuration stored locally. These two factors do not change with the mesh scale.
 
-It is also worth mentioning that enabling permissive mode will cause data plane to use much more memory, as each sidecar needs to keep the connection configuration between all other services. The chart below shows the sidecar memory usage when 200 and 400 pods are added in a permissive-mode enabled mesh. Compared to the non-permissive mode, the memory usage is more than doubled. Similar things happen on the control plane.
+### Other Scenarios
+
+#### Permissive mode
+
+When permissive mode is enabled in OSM, services are allowed to communicate with any other services in the mesh without additional traffic policies. This is implemented by adding an allow-all wildcard RBAC policy in the inbound listener and all known services as outbound clusters in the Envoy sidecar. Permissive model still enable mTLS authentication between services and requires the services be added inside the mesh. The same load test described above is performed on a service mesh with permissive mode enabled.
+
+On the control plane, the CPU and memory usage both increase compared to non-permissive mode.
+
+On the data plane, there is no obvious difference in terms of CPU usage. However, the memory usage is much higher in permissive mode. This is expected as more outbound cluster information is stored in the sidecar. The chart below shows the sidecar memory usage when 10 and 20 services are added in permissive mesh and non-permissive mesh respectively. Compared to the non-permissive mode, the memory usage is more than doubled.
 
 <p align="center">
   <img src="/docs/images/perf/perm-memory.png" width="650"/>
 </p>
+
+The request latency from load generator to workloads is lower in permissive mode. This reduction is due to the RBAC policy filter path being a single wildcard filter as described above. This filter path is shorter than non-permissive mode and there is less rule matching operations for incoming requests. In our test, we observe the P99 latency drops to around 80% in permissive mode. The more SMI traffic policies added to services in non-permissive mode, the larger request time overhead we expect. Below is the latency comparison between permissive and non-permissive mode, tested with 250 per pod RPS:
+
+<p align="center">
+  <img src="/docs/images/perf/latency-perm-vs-non-perm.png" width="650"/>
```

Comment:
similarly these are untenable results. i'm virtually positive that there is a bottleneck unrelated to OSM that is yielding these results.

Why can't we run this on a large cluster?